session/session.go

```diff
@@ -252,9 +253,9 @@ func (s *session) OpenCh(pctx context.Context, openingBalInfo perun.BalInfo, app
 	}
 
 	sanitizeBalInfo(openingBalInfo)
-	parts, err := retrieveParts(openingBalInfo.Parts, s.contacts)
+	parts, err := retrieveParts(openingBalInfo.Parts, s.idProvider)
```

Comment:
```suggestion
	partIDs, err := retrievePartIDs(openingBalInfo.Parts, s.idProvider)
```
As we renamed Peer to PeerID, renaming these variables accordingly.
Please update the variable names in other places of usage as well.

`openingBalInfo.Parts` can remain the same, as it will contain only the `aliases` of the channel participants.

Reply:
Change the `parts` to `partIDs` in line no 273 as well.

Reply:
Keeping `parts` itself everyplace instead of `partIDs`, since parts also looks fine.